GetEycaStatus/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "api/v1/cgn/status/{fiscalcode}",
```

Comment:
Isn't it "api/v1/cgn/eyca/status/{fiscalcode}"?

Reply:
Sure it is

Reply:
Done in [63360ab](https://github.com/pagopa/io-functions-cgn/pull/25/commits/63360abb4958e58b6b0c5a10fb9d0c8b51b91d64)